src/System.Runtime.Intrinsics.X86/ref/System.Runtime.Intrinsics.X86.cs

```diff
@@ -169,8 +169,7 @@ public static class Avx
         public static Vector128<double> Permute(Vector128<double> value, byte control) { throw null; }
         public static Vector256<float> Permute(Vector256<float> value, byte control) { throw null; }
         public static Vector256<double> Permute(Vector256<double> value, byte control) { throw null; }
-        public static Vector256<float> Permute2x128(Vector256<float> left, Vector256<float> right, byte control) { throw null; }
-        public static Vector256<double> Permute2x128(Vector256<double> left, Vector256<double> right, byte control) { throw null; }
+        public static Vector256<T> Permute2x128<T>(Vector256<T> left, Vector256<T> right, byte control) where T : struct { throw null; }
         public static Vector128<float> PermuteVar(Vector128<float> left, Vector128<float> mask) { throw null; }
```

Comment:
Should this be called `PermuteVariable`? For consitency with `PermuteVar`. 

Or do these two methods need the Var/Variable suffix at all? Would overload be sufficient?

Reply:
We are using the `Variable` suffix only for `v`-suffixed instructions, e.g., `BlendVariable` -> `vblendvp*`, `ShiftLeftLogicalVariable`->`vpsllv*`, etc.
`PermuteVar` and `PermuteVar8x32` is a special case that will generate `vpermilp*` and `vperm*`, which breaks the above convention, so it is following C++ Intel intrinsic naming. 
We can change it to `Variable` suffix, I have no strong preference here.